### PR TITLE
[URGENT] Add section for verifying up-to-date CAs

### PR DESCRIPTION
**Pull request by brianhlin**
Hi everyone, I'd like to reference this section in an announcement to all OSG sites and VOs today. Please review this ASAP!

### CODE REVIEW

File: docs/common/ca.md [hunk 1]
```diff
@@ -104,7 +104,64 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
 Verifying CA Certificates
 -------------------------
 
-After installing the CA certificates, they can be verified with the following command:
+### Are up-to-date ###
```
Comment (matyasselmeci) [inline docs/common/ca.md:107]: This line looks weird. How about making it an h2 with `## Verifying CA certificates are up-to-date`, and moving it below the `Post-installation` section? Perhaps also promote `Post-installation` to an h2 and change it to `## Verifying CA certificates are properly installed`?

File: docs/common/ca.md [hunk 2]
```diff
@@ -104,7 +104,64 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
...
+1.  Retrieve the versions of the most recently released
+    [IGTF CA certificates](https://repo.opensciencegrid.org/cadist/index.html) and
+    [OSG CA certificates](https://repo.opensciencegrid.org/cadist/index-new.html)
+
+1.  Determine which of the three CA certificate installation methods you are using:
+
+    1.  Check for an existing RPM installation (only one of the following RPM packages may be installed at a time):
+
+            :::console
+            # rpm -q igtf-ca-certs osg-ca-certs
```
Comment (matyasselmeci) [inline docs/common/ca.md:123]: Something along the lines of:

Run this command to find out which of the following items applies to you:
```suggestion
            # rpm -q igtf-ca-certs osg-ca-certs empty-ca-certs osg-ca-scripts
```
if `igtf-ca-certs` or `osg-ca-certs` are installed, read option 1. If `osg-ca-scripts` is installed, read option 2. If `empty-ca-certs` is installed, read option 3.

**Comment by djw8605** [comment]
Are the sections going to be numbered by letters?  2b?  GitHub seems to render it as i, ii, iii ...

**Comment by brianhlin** [comment]
Yeah, MkDocs renders numbered sub-bullets with a, b, c

File: docs/common/ca.md (hunk 1 above)
Comment (brianhlin) [reply docs/common/ca.md]: How about now?

File: docs/common/ca.md [hunk 3]
```diff
@@ -101,10 +101,10 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
 |:---------------|:-----------------------|:-------------------------------------|
 | [cilogon-openid](https://ca.cilogon.org/policy/openid) | cilogon-openid-ca-cert | `yum install cilogon-openid-ca-cert` |
 
-Verifying CA Certificates
--------------------------
+Verifying CA Certificate
```
Comment (matyasselmeci) [inline docs/common/ca.md:104]: Why drop the plural?

**Comment by brianhlin** [comment]
CI failures unrelated, merging

**Pull request merged**